### DISCUSSION TITLE
Circular Map

### DISCUSSION THREAD

**Discussion by jhein05** [hassio-addons/addon-zwavejs2mqtt#219] [Ideas]
Especially with lots of nodes, I feel like a circular map would be the best way of visualizing the network, because the linear one becomes really wide. I attached an example (from Nagios).
![image](https://user-images.githubusercontent.com/5014124/126049724-64e1b419-60bf-4d51-bb31-67a6fbee8bfa.png)

**Comment by frenck** [comment]
Hi there!

Thanks for your suggestion, however, this is the project that creates/maintains the add-on, not the Z-Wave JS 2 MQTT project itself.

If you have suggestions for the software running inside an add-on, you should request that upstream.

<https://github.com/zwave-js/zwavejs2mqtt>

Thanks! 👍

**Reply by jhein05** [reply]
> If you have suggestions for the software running inside an add-on, you should request that upstream.
> https://github.com/zwave-js/zwavejs2mqtt

Good point :)
